src/components/Pricing/Test/SimilarProducts.tsx

```diff
@@ -0,0 +1,123 @@
+import React from "react"
+import { section, SectionHeader } from './Sections'
+import { usePlatform } from '../Platform/usePlatform'
+import useProducts from '../Products'
+import { IconCheck, IconX } from '@posthog/icons'
+import Logo from "components/Logo"
+import { StaticImage } from "gatsby-plugin-image"
+
+const comparison = [
+  {
+    "name": "Amplitude",
+    "logo": <StaticImage src={`./images/icon-amp.png`} className="" placeholder="blurred" alt="Amplitude" />,
+    "products": {
+      "ProductAnalytics": true,
+      "WebAnalytics": false,
+      "SessionReplay": true,
+      "FeatureFlags": false,
+      "AbTesting": true,
+      "Surveys": false
+    }
+  },
+  {
+    "name": "Mixpanel",
+    "logo": <StaticImage src={`./images/icon-mp.png`} className="" placeholder="blurred" alt="Mixpanel" />,
+    "products": {
+      "ProductAnalytics": true,
+      "WebAnalytics": false,
+      "SessionReplay": false,
+      "FeatureFlags": false,
+      "AbTesting": false,
+      "Surveys": false
+    }
+  },
+  {
+    "name": "Heap",
+    "logo": <StaticImage src={`./images/icon-heep.png`} className="h-full" height={32} placeholder="blurred" alt="Mixpanel" />,
+    "products": {
+      "ProductAnalytics": true,
+      "WebAnalytics": false,
+      "SessionReplay": true,
+      "FeatureFlags": false,
+      "AbTesting": false,
+      "Surveys": false
+    }
+  },
+  {
+    "name": "Pendo",
+    "logo": <StaticImage src={`./images/icon-pnd0.png`} className="" placeholder="blurred" alt="Mixpanel" />,
+    "products": {
+      "ProductAnalytics": true,
+      "WebAnalytics": false,
+      "SessionReplay": true,
+      "FeatureFlags": false,
+      "AbTesting": false,
+      "Surveys": false
+    }
+  },
+  {
+    "name": "FullStory",
+    "logo": <StaticImage src={`./images/icon-fs.png`} className="" height={32} placeholder="blurred" alt="Mixpanel" />,
+    "products": {
+      "ProductAnalytics": true,
+      "WebAnalytics": false,
+      "SessionReplay": true,
+      "FeatureFlags": false,
+      "AbTesting": false,
+      "Surveys": false
+    }
+  },
+  {
+    "name": "PostHog",
+    "logo": <div className="-ml-6"><Logo className="h-5" /></div>,
+    "products": {
+      "ProductAnalytics": true,
+      "WebAnalytics": true,
+      "SessionReplay": true,
+      "FeatureFlags": true,
+      "AbTesting": true,
+      "Surveys": true
+    }
+  }
+]
+
+export const SimilarProducts = () => {
+
+  return (
+    <section className={`${section} mb-12 mt-8 md:px-4`}>
+      <SectionHeader>
+        <h3 className="mb-2">Compare to similar products</h3>
```

Comment:
We should link these to our actual comparison blogs

Reply:
Just hesitant to add more links _away_ from the pricing page